### DISCUSSION TITLE
Neither Ardour nor Carla recognize N.A.M after successful build

### DISCUSSION THREAD

**Discussion by OnlyANegative** [mikeoliphant/neural-amp-modeler-lv2#45] [General]
Hello, 

My device is a Macbook Pro early 2015 13inch running Manjaro Linux 6.4.0_rt6-1. 

My problem is that after the output of the _cmake .. -DCMAKE_BUILD_TYPE="Release"_ command appeared successful neither Carla nor Ardour can see the lv2 plugin after scanning. Ardour doesn't report N.A.M at all, not even in the error list.

I also tried loading the .so file with Carla from its /build/neural_amp_modeler.lv2 folder. The result was the following error: 

"Failed to load file

Could not find the VST2 main entry in the plugin library"

Below is the output of cmake & make:
![neural-amp-modeler](https://github.com/mikeoliphant/neural-amp-modeler-lv2/assets/139655553/5638fb9d-9f58-45d8-a431-f25c8b7f8bfd)

I apologize if this isn't the correct section for beginner troubles. I'd also like to apologize if this is an incredibly simple issue to solve.

**Comment by mikeoliphant** [comment]
Are you running carla from any kind of virtual environment?

See issue https://github.com/mikeoliphant/neural-amp-modeler-lv2/issues/43

**Reply by OnlyANegative** [reply]
Thanks for the reply, 

No I am not. Rather, I don't know how to setup a virtual environment so I assume it's not something I can do by accident.

**Comment by alanpasi** [comment]
For Linux support, there is an LV2 plugin available:
https://github.com/mikeoliphant/neural-amp-modeler-lv2.

Em sáb., 15 de jul. de 2023 19:38, Sam Smith ***@***.***>
escreveu:

**Reply by OnlyANegative** [reply]
Thank you for the reply,

Sorry, I wasn't clear but the LV2 plugin you linked is what I tried building and am inquiring about. Unless there are separate releases for Archlinux/Manjaro I'm using the correct one.

**Comment by donnekgit** [comment]
Just seconding this.  I've just built off the latest code, which went fine, but Ardour (6.9) can't see the plugin.  I'm using AVLinux-MXE-21.

**Comment by mikeoliphant** [comment]
This issue may be related: https://github.com/mikeoliphant/neural-amp-modeler-lv2/issues/52

**Reply by donnekgit** [reply]
Recompiled, and Ardour sees it now.  But trying to insert it on the track gives a message that it could not be loaded.  Unfortunately, there's no reference to it in the Ardour log, so it's unclear why it can't be loaded...
